<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiTeacher</class>
 <widget class="QWidget" name="QtGuiTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiTeacher</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>133</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit"/>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>Grade</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
